packages/core-svelte/src/components/TextDropdownComponent.svelte

```diff
@@ -336,6 +333,26 @@
         }
     };
 
+    const onFocusOut = () => {
+        LOGGER.log("onFocusOut " + id, "focus");
+    };
+
+    const onFocusOutText = () => {
+        LOGGER.log("onFocusOutText " + id, "focus");
+        if (isEditing) {
+            isEditing = false;
+        }
+    };
+
+    const onClick = () => {
```

Comment:
This component should never receive an on-click. Clicks should be handled by either the textcomponent, or the dropdown component.